app/components/EditableLabel/EditableLabel.js

```diff
@@ -146,10 +146,11 @@ export default class EditableLabel extends React.Component {
       ? this.state.text
       : this.props.labelPlaceHolder || DEFAULT_LABEL_PLACEHOLDER;
     return (
-      <div className={this.props.containerClassName}>
+      <div className={this.props.containerClassName}
+        onClick={this.handleFocus}
+      >
```

Comment:
```suggestion
     <div className={this.props.containerClassName} onClick={this.handleFocus}>
```

Looks good to me, but could you please format it properly?